triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/RunnerConfig.scala

```diff
@@ -249,6 +249,16 @@ object RunnerConfig {
           throw new IllegalArgumentException(s"Unsupported logging encoder $other")
       }
 
+    opt[Long]("max-batch-size")
+      .optional()
+      .text(
+        s"maximal number of message processed between two high-level rule triggers. Defaults to ${DefaultTriggerRunnerConfig.maximumBatchSize}"
```

Comment:
Nit pick: maybe write "maximum number of messages..." instead?

Reply:
:+1: